### PR TITLE
Feature/list notes

### PR DESCRIPTION
**Pull request by jamarob**
See [User Story](https://github.com/jamarob/capstone-project/issues/1)

### CODE REVIEW

**Comment by arisslin** [comment]
GlobalStyle.js evtl in einen Ordner 'common'

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,30 @@
-import React from "react";
+import React from 'react'
+import {
+  BrowserRouter as Router,
+  Redirect,
+  Route,
+  Switch,
+} from 'react-router-dom'
+import NotesList from './components/NotesList'
+import GlobalStyle from './GlobalStyle'
```
Comment (arisslin) [inline src/App.js]: Maybee GlobalStyle.js in a folder named 'common'?

File: src/components/Date.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export default function({ millis }) {
```
Comment (arisslin) [inline src/components/Date.js]: millis to milliseconds or ms

File: src/components/Note.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import styled from 'styled-components'
+import Date from './Date'
```
Comment (arisslin) [inline src/components/Note.js]: If you need Date later in another component than i think it is a good idea to give it it's own file. If not i think you can add it to your Note component or create a helper function of it.

File: src/App.js (hunk 1 above)
Comment (jamarob) [reply src/App.js]: `GlobalStyle.js`is only needed in `App.js` and not shared across other components. I will keep it at at the same level for now.

File: src/components/Note.js (hunk 3 above)
Comment (jamarob) [reply src/components/Note.js]: I agree. It's only needed in `Note.js`. I will put it in `Note.js`.

**Pull request merged**